app/assets/javascripts/submissions.coffee

```diff
@@ -270,7 +270,7 @@ class TrialSubmission extends Submission
         @$('.uploaded-layout .delete-layout-upload').attr(href: data.result.delete_url)
 
         @$('.uploaded-layout .layout-image').prop(src: data.result.small_file_url)
-        @$('.uploaded-layout .layout-image').parent().prop(href: data.result.original_file_url)
+        @$('.uploaded-layout .layout-image').parent('a').prop(href: data.result.original_file_url)
```

Comment:
It definitelly works for me without this change.

2016-06-04 16:06 GMT+02:00 Tomasz Gubała <notifications@github.com>:

> In app/assets/javascripts/submissions.coffee
> <https://github.com/eSpectrum-IT/brassica/pull/524#discussion_r65802931>:
>
> > @@ -270,7 +270,7 @@ class TrialSubmission extends Submission
> >          @$('.uploaded-layout .delete-layout-upload').attr(href: data.result.delete_url)
> >
> >          @$('.uploaded-layout .layout-image').prop(src: data.result.small_file_url)
> > -        @$('.uploaded-layout .layout-image').parent().prop(href: data.result.original_file_url)
> > +        @$('.uploaded-layout .layout-image').parent('a').prop(href: data.result.original_file_url)
>
> Well, it didn't work without it. That's why I did that fix.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eSpectrum-IT/brassica/pull/524/files/8577b3a75c449dcc2041b914f48f772ef38128da..4af566e069cb847caf6622cff3018785129d7bb7#r65802931>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAA2yeO3zZh16vOC0v_yq7_h0FoSYV2xks5qIYZ3gaJpZM4Is_Pt>
> .
>



-- 
Tomasz Szymczyszyn